dev/source/docs/mavlink-get-set-home-and-origin.rst

```diff
@@ -15,16 +15,14 @@ The "EKF origin" is the location that the EKF (aka AHRS) uses for internal calcu
 
 Whenever the Home or EKF origin is updated the vehicle will send a `HOME_POSITION <https://mavlink.io/en/messages/common.html#HOME_POSITION>`__ or `GPS_GLOBAL_ORIGIN <https://mavlink.io/en/messages/common.html#GPS_GLOBAL_ORIGIN>`__ message (respectively) on all active mavlink channels.
 
-The home will also be sent in response to a `MAV_CMD_GET_HOME_POSITION <https://mavlink.io/en/messages/common.html#MAV_CMD_GET_HOME_POSITION>`__ sent within a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ message.
+The home will also be sent in response to a `MAV_CMD_GET_HOME_POSITION <https://mavlink.io/en/messages/common.html#MAV_CMD_GET_HOME_POSITION>`__ sent within a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ or `COMMAND_INT <https://mavlink.io/en/messages/common.html#COMMAND_INT>`__ message.
 
 .. _mavlink-get-set-home-and-origin_set_home_position:
 
-SET_HOME_POSITION
------------------
+MAV_CMD_DO_SET_HOME within COMMAND_LONG
+---------------------------------------
 
-Sets the location that the vehicle will return to and land on when in RTL mode. The location is normally set automatically each time the vehicle is armed if it has not already been explicitly set by the operator.
-
-The message definition can be found `here <https://mavlink.io/en/messages/common.html#SET_HOME_POSITION>`__
+Set the home location by sending a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command and parameter fields set as specified for the `MAV_CMD_DO_SET_HOME <https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_HOME>`__ command.
```

Comment:
Preference is for `COMMAND_INT` as it provides better precision for the location.

Reply:
OK, i've moved the COMMAND_INT method above the COMMAND_LONG method and added a little note about accuracy.  Thanks for caring!